### PR TITLE
Feature/font v2

### PR DESCRIPTION
**Pull request by jperezgonzal**
## Description
Se migró la Font desde android-ui a andes.

## Checks
Are you sure you followed all those steps? In such case, let's say with me "I solemnly declare that ...":
- [ ] I have coded an example inside the Demo App,
- [ ] I have added proper tests,
- [ ] I have modified the [Changelog](https://github.com/mercadolibre/fury_andesui-android/blob/develop/CHANGELOG.md) file,
- [ ] I have updated GitHub wiki,
- [ ] I have the explicit approval of one or more members of the UX Team,
- [ ] I have updated the version in gradle.properties file.

## Change type
This is easy, let us know what this code is about:
- [x] This code adds a new component
- [ ] This code includes improvements to an existent component
- [ ] This code improves or modifies ONLY the Demo App.

## Check common errors
Whether you are an Android or iOS developer and if you're still there then we'll give you a present:
https://proandroiddev.com/how-to-maximize-androids-ui-reusability-5-common-mistakes-cb2571216a9f

### CODE REVIEW

**Comment by furyGorgoryAdmin** [comment]
### Informative Menu <h2>  ### 

### Use any of these tags in a pull request description or leave a comment ### 
| Tag | Description |
|---|---|
| **[canary]**| Creates a build that can only be deployed with canary strategy. | 
| **[build]** |  Creates version or build from a branch that has been tagged as "no releasable". | 
| **[name:version-name]** |Specify the name of the build. For example: [build] [name:1.2.3-name] or [canary] [name:1.2.3-name].|
| **[major]** | To automatically increase the first position of the version. For example: [build] [major] or [canary] [major].|
| **[minor]** | To automatically increase the second position of the version. For example: [build] [minor] or [canary] [minor]. |
| **[patch]** | To automatically increase the third position of the version. For example: [build] [patch] or [canary] [patch].|  

[For more information about ci-cd](http://docs.furycloud.io/?lang=esp&section=businessassurance)
[For more information about semver](https://semver.org/#summary)

File: components/src/main/java/com/mercadolibre/android/andesui/font/TypefaceSpanCompat.kt [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        }
+        paint.typeface = typeFace
+    }
+
+    private fun getOldStyle(oldTypeface: Typeface?): Int {
+        return oldTypeface?.style ?: 0
+    }
+
+    companion object {
+        private const val SKEW_X = -0.25f
```
Comment (jperezgonzal) [inline components/src/main/java/com/mercadolibre/android/andesui/font/TypefaceSpanCompat.kt:46]: Esta clase ya estaba en **ui**, yo simplemente la pasé para andes. Esta clase es usada para los TypefaceSpan, pero desconozco exactamente que es lo que hace esto.

File: components/src/main/java/com/mercadolibre/android/andesui/font/TextViewExtension.kt [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package com.mercadolibre.android.andesui.font
+
+import android.graphics.Paint
+import android.widget.TextView
+
+fun TextView.setTypeface(font: Font) {
+    TypefaceHelper.setTypeface(this, font)
+}
+
+fun TextView.setTypeface(paint: Paint, font: Font) {
```
Comment (juanimoli) [inline components/src/main/java/com/mercadolibre/android/andesui/font/TextViewExtension.kt:10]: te va agregarle documentacion a estas extensions?

File: components/src/main/java/com/mercadolibre/android/andesui/font/TypefaceSpanCompat.kt (hunk 1 above)
Comment (juanimoli) [reply components/src/main/java/com/mercadolibre/android/andesui/font/TypefaceSpanCompat.kt:46]: como podriamos mejorar esto? quizas mejor que copiar y pegar podemos agregarle documentacion, que opinas?

File: components/src/main/java/com/mercadolibre/android/andesui/font/TypefaceSpanFactory.kt [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+    @JvmStatic
+    fun create(typeface: Typeface): TypefaceSpan {
+        return if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P) {
+            TypefaceSpan(typeface)
+        } else TypefaceSpanCompat(typeface)
+    }
+
+    @JvmStatic
+    fun create(context: Context, font: Font): TypefaceSpan? {
```
Comment (juanimoli) [inline components/src/main/java/com/mercadolibre/android/andesui/font/TypefaceSpanFactory.kt:21]: idem aca, unas docus no estaria mal

**Pull request merged**